INSTALLATION.md

```diff
@@ -25,8 +26,8 @@ This document provides instructions on how to set up and start a running instanc
   - [Debugging tests](#debugging-tests)
   - [Linting code files](#linting-code-files)
   - [Husky for Git Hooks](#husky-for-git-hooks)
-    - [pre-commit hook](#pre-commit-hook)
-    - [post-merge hook](#post-merge-hook)
+      - [pre-commit hook](#pre-commit-hook)
```

Comment:
Why change this file ?

Reply:
Our new commit githooks now automatically regenerate the table of contents of the markdown files in the root directory across most repositories.